src/generated/gitInfo.ts

```diff
@@ -1 +1 @@
-export default { commitHash: "4aaf371", version: "1.0.0-rc2-21-g4aaf371" };
+export default { commitHash: "d13cf63", version: "1.0.0-rc2-10-gd13cf63" };
```

Comment:
`src/generated/gitInfo.ts` shouldn't be committed to the repo anymore. It was added to the `.gitignore` file. Could you please check if `git` is not caching this locally for you? :pray: This file was causing too many unnecessary conflicts because it's autogenerated for every commit.